src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.cs

```diff
@@ -232,6 +233,69 @@ public void Dispose_UnregisterFromSelectionChangedEvent()
             selectionService.VerifyRemove(x => x.SelectedIssueChanged -= It.IsAny<EventHandler>(), Times.Once());
         }
 
+        [TestMethod]
+        public void ShowVisualizationPaneCommand_CanExecute_NullParameter_False()
+        {
+            var menuCommandService = new Mock<IMenuCommandService>();
+            var testSubject = CreateTestSubject(menuCommandService: menuCommandService.Object);
+
+            VerifyCommandExecution(testSubject.ShowVisualizationPaneCommand, null, false);
+
+            menuCommandService.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public void ShowVisualizationPaneCommand_CanExecute_ParameterIsNotTaintViewModel_False()
+        {
+            var menuCommandService = new Mock<IMenuCommandService>();
+            var testSubject = CreateTestSubject(menuCommandService: menuCommandService.Object);
+
+            VerifyCommandExecution(testSubject.ShowVisualizationPaneCommand, "something", false);
+
+            menuCommandService.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public void ShowVisualizationPaneCommand_CanExecute_ParameterIsTaintViewModel_True()
+        {
+            var menuCommandService = new Mock<IMenuCommandService>();
+            var testSubject = CreateTestSubject(menuCommandService: menuCommandService.Object);
+
+            VerifyCommandExecution(testSubject.ShowVisualizationPaneCommand, Mock.Of<ITaintIssueViewModel>(), true);
+
+            menuCommandService.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public void ShowVisualizationPaneCommand_Execute_LocationNavigated()
+        {
+            var menuCommandService = new Mock<IMenuCommandService>();
+
+            var viewModel = new Mock<ITaintIssueViewModel>();
```

Comment:
FYI if you are creating a mock just to get a valid object (i.e. if the default values are ok and you don't need to verify the mock afterwards) you can use `Mock.Of<ITaintIssueViewModel>`.  It's slightly shorter, but more importantly it makes it clear that the object won't been checked later.

Reply:
Ah; I see you've used `Mock.Of<>` above. It's good to be consistent.